src/system.h

```diff
@@ -906,6 +906,12 @@ typedef struct {
     const Char *    cookie;
 } StructGVarFunc;
 
+#define GAP_GVAR_FUNC_TABLE_ENTRY(srcfile, name, nparam, params) \
+  {#name, nparam, \
+   params, \
+   (GVarFunc)name, \
+   srcfile ":Func" #name }
```

Comment:
But even if you make `Func` enforced prefix, this is still wrong. You are using raw `name` as the name of the C function in the line `(GVarFunc)name`, then attaching a `Func` in the next line, so if the code compiled, `name` must already contain `Func`, and you'd print `FuncFunc...`.

There is some (slightly messy) macro-foo which would let you attach the `Func` to `name` as the name of a C function. I can dig it out.